### PR TITLE
Updated the pull request template

### PR DESCRIPTION
**Pull request by DamianX**
:cl:
 * rscadd: Did stuff!
 * rscdel: did other stuff!

### CODE REVIEW

**Comment by calzilla1** [comment]
Why

**Comment by PrimeDSS13** [comment]
what did damian mean by this

**Comment by SonixApache** [comment]
why

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -91,4 +88,4 @@ An example changelog is attached to this PR for your convenience:
 -->
 :cl:
  * rscadd: Did stuff!
- * rscdel: did other stuff!
\ No newline at end of file
+ * rscdel: did other stuff!
```
Comment (Exxion) [inline .github/PULL_REQUEST_TEMPLATE.md:91]: What's even changed on this line

**Comment by Exxion** [comment]
Also funny meme but I'm afraid someone will merge it as a joke so I have to do this
Sorry

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (Zth--) [reply .github/PULL_REQUEST_TEMPLATE.md:91]: EOL

Comment (Exxion) [reply .github/PULL_REQUEST_TEMPLATE.md:91]: How did that sneak in? Our EOLs should be standardized

**Comment by DamianX** [comment]
we live in a society

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (DamianX) [reply .github/PULL_REQUEST_TEMPLATE.md:91]: I guess the template is older than `.editorconfig`

**Pull request closed without merging**